db_test.go

```diff
@@ -487,6 +491,9 @@ func Test_DB_ColdStart(t *testing.T) {
 			db, err = c.DB(context.Background(), sanitize(t.Name()))
 			require.NoError(t, err)
 
+			_, err = db.Table(sanitize(t.Name()), config)
```

Comment:
The point of the cold start test was to ensure we could read data after restarting without having to reconfigure the tables schema. This test should just be deleted as we no longer hold to that.

Reply:
Makes sense!